config/betamocks/services_config.yml

```diff
@@ -537,7 +537,7 @@
 
 #VANotify
 - :name: 'VANotify'
-  :base_uri: <%= "#{URI(Settings.vanotify.client_url).host}:#{URI(Settings.vanotify.client_url).port}" %>
+  :base_uri: 'http://fakeapi.com'
```

Comment:
```suggestion
  :base_uri: <%= Settings&.vanotify&.client_url ? "#{URI(Settings.vanotify.client_url).host}:#{URI(Settings.vanotify.client_url).port}" : 'http://fakeapi.com' %>
```

Reply:
i changed it in the wrong spot, going to push the better fix